docs/pages/production.mdx

```diff
@@ -42,6 +42,7 @@ Teleport services listen on several ports. This table shows the default port num
 | `3025` | Auth | SSH port used by the Auth Service to serve its Auth API to other nodes in a cluster. | Allow inbound connections from all cluster nodes. | Allow outbound traffic to cluster nodes. |
 | `3024` | Proxy | SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server. | {/* TODO */} | {/* TODO */} |
 | `3026` | Kubernetes | Port used for `kubectl` to access Teleport | {/* TODO */} | {/* TODO */} |
+| `3036` | DB | Teleport MySQL proxy listen address. | {/* TODO */} | {/* TODO */} |
```

Comment:
Added missing proxy port number which is referenced elsewhere but not listed with the other commonly used ports.